helm/resource_release.go

```diff
@@ -45,7 +52,33 @@ func resourceRelease() *schema.Resource {
 			"repository": {
 				Type:        schema.TypeString,
 				Optional:    true,
-				Description: "Repository where to locate the requested chart. If is an URL the chart is installed without installing the repository.",
+				Description: "Repository where to locate the requested chart. If is a URL the chart is installed without installing the repository.",
+			},
+			"repository_key_file": {
```

Comment:
Should the repository settings be moved into their own block of settings like kubernetes settings are in the provider?